modules/apps/document-library/document-library-item-selector-web/src/main/java/com/liferay/document/library/item/selector/web/internal/display/context/DLItemSelectorViewDisplayContext.java

```diff
@@ -368,16 +377,28 @@ private Long _getFileEntryTypeId() {
 			infoItemItemSelectorCriterion.getItemSubtype());
 	}
 
-	private long _getFolderId() {
-		if (_folderId != null) {
-			return _folderId;
+	private long _getFolderIdFromRequestParameters(
```

Comment:
Just a minor thing: I'd prefer to keep the private method name simply as `_getFolderId`. Qualifying the name would make sense if there where different sources from where the folder ID was read, but it is always read from the request. I think it is redundant.

But I don't think this is a blocker, so I've approved the pr.

Reply:
Change done. I prefer to change the name now and not be asked later when merging to master 😅